cmd/shell.go

```diff
@@ -0,0 +1,151 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/puppetlabs/wash/cmd/internal/server"
+	cmdutil "github.com/puppetlabs/wash/cmd/util"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func writeAlias(path, subcommand string) error {
+	f, err := os.OpenFile(path, os.O_CREATE|os.O_WRONLY, 0750)
```

Comment:
Oh I see, you're creating a script that just invokes the subcommand. A quick comment explaining that would be nice.